aten/src/ATen/native/cuda/RowwiseScaledMM.cu

```diff
@@ -74,6 +74,32 @@ using DtypeAccum = float;
 using DtypeEpilogue = float;
 using DtypeOutput = cutlass::bfloat16_t;
 
+template <typename T>
+struct identity {
```

Comment:
noob q: is it better to reuse the ones defined in cutlass: https://github.com/NVIDIA/cutlass/blob/f7b19de32c5d1f3cedfc735c2849f12b537522ee/include/cutlass/epilogue/thread/activation.h#L57
or 
https://github.com/NVIDIA/cutlass/blob/f7b19de32c5d1f3cedfc735c2849f12b537522ee/include/cute/algorithm/functional.hpp#L46
?

Reply:
Ah, thanks, I was looking for it here and couldn't find it! https://github.com/NVIDIA/cutlass/blob/f7b19de32c5d1f3cedfc735c2849f12b537522ee/include/cutlass/functional.h